docker-compose.yml

```diff
@@ -18,6 +18,11 @@ services:
     depends_on:
       - "db"
       - "memcached"
+    build:
+      args:
+        - TRANSCENDENCE_APP_DATA=${TRANSCENDENCE_APP_DATA:-/var/lib/transcendence-app}
```

Comment:
This syntax is to set a default value in case this variable is undefined

Reply:
Why do we even have to set it if there is a default and that is the default we all need to set it to?

Reply:
The arg from here is an option if someone prefers to store the data in another directory inside the Docker container. Since it's not a confidential setting, we set it by default to `/var/lib/transcendence-app`. It's similar to how PostgreSQL has a `PGDATA` variable which defines where it stores the data.

With Docker, the application works without setting this variable in the `.env` file of transcendence-app because there is the `ENV` directive in the Dockerfile, and Docker exposes it to the container. But if someone wants to run the application without Docker, this variable is required. I hope it makes sense.